### PR TITLE
[TIMOB-19170] iOS: ListView backgroundGradient not full row size

### PR DESCRIPTION
**Pull request by sriks**
JIRA https://jira.appcelerator.org/browse/TIMOB-19170

### CODE REVIEW

File: iphone/Classes/TiUIListItem.m [hunk 1]
```diff
@@ -116,6 +116,10 @@ - (void)layoutSubviews
 		[_proxy layoutChildren:NO];
 #endif
 	}
+	
+	if (gradientLayer) {
+		[gradientLayer setFrame:[self bounds]];
```
Comment (AngelkPetkov) [inline iphone/Classes/TiUIListItem.m]: Fix indentation level please!

**Comment by AngelkPetkov** [comment]
FT and CR approved. Will merge ones comment is addressed.Thank you!

**Comment by sriks** [comment]
Comment addressed.

**Comment by AngelkPetkov** [comment]
Thanks! Merging

**Pull request merged**